app-backend/database/src/main/scala/com/azavea/rf/database/tables/Projects.scala

```diff
@@ -86,6 +86,42 @@ object Projects extends TableQuery(tag => new Projects(tag)) with LazyLogging {
     }
   }
 
+  /** Get the AOIs belonging to a project. */
+  def listAOIs(
+    projectId: UUID,
+    pageRequest: PageRequest,
+    user: User
+  )(implicit database: DB): Future[PaginatedResponse[AOI]] = {
+    // TODO: This should be a join?
```

Comment:
I think this makes sense for here since like you said there is only ever going to be one project.